packages/itwin/grouping-mapping-widget/src/widget/components/CustomCalculationAction.tsx

```diff
@@ -225,6 +226,13 @@ export const CustomCalculationAction = ({
               validator.showMessageFor("name");
             }}
           />
+          <Alert
+            type='informational'
+            clickableText='Click here.'
+            clickableTextProps={{ href: "https://developer.bentley.com/apis/insights/operations/create-customcalculation/", target: "_blank", rel: "noreferrer" }}
```

Comment:
@Guillar1, is it okay for this url to be like this? Maybe we need an integration test for this to notice if this link breaks for any reason.